src/flow-hash.c

```diff
@@ -878,20 +878,17 @@ Flow *FlowGetExistingFlowFromHash(FlowKey *key, const uint32_t hash)
  */
 static Flow *FlowGetUsedFlow(ThreadVars *tv, DecodeThreadVars *dtv)
 {
-    uint32_t idx = SC_ATOMIC_GET(flow_prune_idx) % flow_config.hash_size;
+    uint32_t idx = SC_ATOMIC_ADD(flow_prune_idx, 25) % flow_config.hash_size;
```

Comment:
Is `25` the number of attempts that will be tried in the `while` loop?

Reply:
Going to change this. It gives each worker a slice of 25 hash rows to check but it will bail after 5. Will update the code to use 5 for both.